bugs/PENDING_REVIEW/0433-BAD-IMAGE-Pixel5-29b7a44e-no-opt-test/reduced_1_result/test.amber

```diff
@@ -0,0 +1,719 @@
+#!amber
+
+SET ENGINE_DATA fence_timeout_ms 60000
+
+SHADER vertex texgen_vert PASSTHROUGH
+
+SHADER fragment texgen_frag GLSL
+#version 430
+precision highp float;
+
+layout(location = 0) out vec4 _GLF_color;
+
+void main()
+{
+ _GLF_color = vec4(
+ gl_FragCoord.x * (1.0 / 256.0),
+ (int(gl_FragCoord.x) ^ int(gl_FragCoord.y)) * (1.0 / 256.0),
+ gl_FragCoord.y * (1.0 / 256.0),
+ 1.0);
+}
+END
+BUFFER default_texture FORMAT B8G8R8A8_UNORM
+
+PIPELINE graphics texgen_pipeline
+  ATTACH texgen_vert
+  ATTACH texgen_frag
+  FRAMEBUFFER_SIZE 256 256
+  BIND BUFFER default_texture AS color LOCATION 0
+END
+
+CLEAR_COLOR texgen_pipeline 0 0 0 255
+CLEAR texgen_pipeline
+RUN texgen_pipeline DRAW_RECT POS 0 0  SIZE 256 256
+
+SHADER vertex reference_vertex_shader PASSTHROUGH
+
+# reference_fragment_shader is derived from the following GLSL:
+# #version 320 es
+# precision highp float;
+#
+# layout(location = 0) out vec4 _GLF_color;
+#
+# layout(set = 0, binding = 0) uniform sampler2D tex;
+#
+# float ReallyApproxNormalizedAtan2(vec2 v)
+# {
+#  float pi2 = 1.0 / (355.0 / 113.0);
+#  if(length(v) < 0.001)
+#   {
+#    return 0.0;
+#   }
+#  vec2 a = abs(v);
+#  float z;
+#  if(a.y > a.x)
+#   {
+#    z = a.x / a.y;
+#   }
+#  else
+#   {
+#    z = a.y / a.x;
+#   }
+#  float th = (0.97 - 0.19 * z * z) * z * pi2;
+#  if(a.y < a.x)
+#   {
+#    th = 0.5 - th;
+#   }
+#  if(v.x < 0.0)
+#   {
+#    th = 1.0 - th;
+#   }
+#  if(v.y < 0.0)
+#   {
+#    th = - th;
+#   }
+#  return th;
+# }
+# vec2 polarize(vec2 coord)
+# {
+#  float pi = (355.0 / 113.0);
```

Comment:
In the variant shader this has been removed. I believe this is because the glsl-fuzz reducer is able to observe that it is never used and get rid of it.

It looks like this line of code shouldn't really be in the reference in the first place. So can you (a) remove it here (I suggest doing so by modifying the sources for this test and re-running the Python script), and (b) do a PR to the GraphicsFuzz repo to remove it from the associated stable shader?